### ISSUE TITLE
CSA badly estimated-- problem with finding the right slices

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#672]
data: batch_processing.sh

~~~
sct_process_segmentation -i t2_seg.nii.gz -p csa -t label/template/MNI-Poly-AMU_level.nii.gz -l 3:4
~~~

output:
~~~
z=0: 31.2230135716 mm^2
z=1: 31.42920418 mm^2
z=2: 31.6349829495 mm^2
z=3: 31.8407617191 mm^2
z=4: 32.0469523275 mm^2
z=5: 32.2538127104 mm^2
z=6: 32.4614413404 mm^2
z=7: 32.6697778513 mm^2
z=8: 32.8790421738 mm^2
z=9: 33.0892839547 mm^2
z=10: 33.2999446857 mm^2
z=11: 33.509868365 mm^2
z=12: 33.7181996852 mm^2
z=13: 33.9243913279 mm^2
z=14: 34.1279244716 mm^2
z=15: 34.3283210314 mm^2
z=16: 34.5247027295 mm^2
z=17: 34.7159464207 mm^2
z=18: 34.9020222358 mm^2
z=19: 35.0837069141 mm^2
z=20: 35.2618172204 mm^2
z=21: 35.4370624914 mm^2
z=22: 35.609596477 mm^2
z=23: 35.7792885367 mm^2
z=24: 35.9463273195 mm^2
z=25: 36.1110759238 mm^2
z=26: 36.2739032974 mm^2
z=27: 36.434455003 mm^2
z=28: 36.5925194613 mm^2
z=29: 36.7487885582 mm^2
z=30: 36.9040888374 mm^2
z=31: 37.0587910711 mm^2
z=32: 37.2131060564 mm^2
z=33: 37.368081666 mm^2
z=34: 37.5247535145 mm^2
z=35: 37.6838046559 mm^2
z=36: 37.8457077696 mm^2
z=37: 38.0105942138 mm^2
z=38: 38.1782416602 mm^2
z=39: 38.3493814699 mm^2
...

Create volume of CSA values...
sct_image -i csa_volume_RPI.nii.gz -setorient AIL -o csa_volume.nii.gz

Generate output files...
  File created: /Users/jcohen/sct_example_data/t2/csa_volume.nii.gz

Find slices corresponding to vertebral levels...
  Check consistency of data size...
    OK!
  Find slices corresponding to vertebral levels...
    19:31
Average CSA across slices...
Mean CSA: 36.0954939316 +/- 0.614764326984 mm^2
~~~

The problem is that the slices DO NOT correspond to these vertebral levels. See below:

![screen shot 2015-11-19 at 5 04 42 pm](https://cloud.githubusercontent.com/assets/2482071/11285841/a86035e4-8edf-11e5-99b1-dec966ec6b20.png)

**Comment by slevyrosetti** [comment]
Indeed, the issue comes from that sct_process_segmentation assumes that the input images are in RPI orientation.
I will add a check in the script.

**Comment by slevyrosetti** [comment]
Actually, the input segmentation is always converted to RPI but not the vertebral labeling file. We only need to convert it.

**Comment by slevyrosetti** [comment]
Fixed in commit #aedc680

**Comment by slevyrosetti** [comment]
And the batch_processing.sh run successfully.
